src/olympia/devhub/templates/devhub/email/submission.txt

```diff
@@ -1,27 +1,23 @@
-Thanks for submitting your {{ addon_name }} add-on to addons.mozilla.org (AMO)! Your add-on listing will go live shortly in our gallery here: {{ detail_url }}
+Subject: Mozilla Add-ons: {{ add-on name }} has been submitted to addons.mozilla.org! 
+
+Thanks for submitting your {{ add-on name }} add-on to addons.mozilla.org (AMO)! Your add-on listing will go live shortly in our gallery here: {{ url }}
 
 Please keep in mind that while your add-on has been screened and approved for listing, other reviewers may look into it in the future and determine that it requires changes or should be removed from the gallery. If that occurs, you will receive a separate notification with details and next steps.
 
-For more information about the review process and policies, please visit https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews
+For more information about the review process and policies, please visit https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews. 
 
 ********
-Attract Users, Stay Updated, & Get in Touch
+Attract Users, Stay Updated, & Get in Touch 
 ********
 
-Add-ons help personalize the web experience for millions of users who have installed Firefox, and we want to help make your users happy! If you have not already done so, we recommend checking out these resources:
-
-
-* Help users discover and install your add-on by learning how to create an appealing listing: https://developer.mozilla.org/Add-ons/Listing
-* Want your add-on featured on AMO? Learn more: https://developer.mozilla.org/Add-ons/AMO/Policy/Featured
-* Join our mailing list to stay up-to-date with news and events relevant to add-on developers, including a subscription to our quarterly newsletter, about:addons: https://addons.mozilla.org/firefox/users/edit#acct-notify
-* Follow the Add-ons Blog to stay up-to-date on the latest news: https://blog.mozilla.org/addons/
-* For more add-on developer resources, please visit MDN web docs: https://developer.mozilla.org/Add-ons
-* Want to ask a question or file a bug report? These are the best ways to contact us: https://developer.mozilla.org/Add-ons#Contact_us
+Add-ons help personalize the web experience for millions of users who have installed Firefox, and we want to help make your users happy! 
 
+To learn how to help users discover your extension, stay up-to-date with news from the add-ons community, or contact the add-ons team, please visit https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution/Resources_for_publishers. 
 
-Happy developing!
+If you are interested in getting more involved with Mozilla and want to help keep the add-ons ecosystem safe and healthy, please consider becoming a volunteer add-on reviewer. You can learn more at https://wiki.mozilla.org/Add-ons/Reviewers. 
 
+Happy developing! 
 
--- 
+--
```

Comment:
Except here! (that's the funny, not obvious part)

Email signatures have to be separated by `-- ` (dash dash whitespace) to work properly. So the whitespace at the end of the line here is actually important. (https://www.ietf.org/rfc/rfc3676.txt Section 4.3. Usenet Signature Convention)